### ISSUE TITLE
New Release?

### ISSUE THREAD

**Issue by hampsterx** [cgrates/cgrates#1368]
Seems ya'll been stuck on 0.9.1 RC8 for a LONG long time!  

https://github.com/cgrates/cgrates/releases/tag/v0.9.1rc7
> rif released this on Sep 22, 2015

RC8 was mentioned as early back as Oct 2015. 

http://www.cgrates.org/ 

Community News
0.9.1rc7 is OUT! 22 September 2015

------

Although it's very clear the project is still in active development (kudos Dan, Teo) when looking at the commit history, those two things really stand out when you first approach the project which is not a great first impression.

**Comment by danbogos** [comment]
hey @hampsterx 

I think it is a matter of vision here ;).

We kinda have changed the feeling regarding releases over time. Not saying we will not do it but reached a small level in our priority list nowadays. Since we treat master as stable (used in production by serious operators), not sure what tagging will solve. More noise - yes, more help from community yes/no? If you want to have a feeling of release, you can download the .deb we publish from time to time and do a cgr-engine -version. You will have different "release" each time.

Regarding activity, I would say we are tagged as "high-activity" project since years (reference here: https://www.openhub.net/p/cgrates). Now this is the part which will bother us if it changes.

On the other hand, if you want to maintain your own releases, it should be pretty straight forward to fork us and tag the master from time to time. 

Hope this helps.

DanB

PS: For discussions and questions, we recommend the google group so others can also benefit, using issues here for issues and feature-requests.

**Comment by hampsterx** [comment]
Hi @danbogos 

I have read more than 1/3 of the google group discussions going back over the last couple of years in a bid to understand how CGRateS works, I created a ticket so it was actionable :)

I once worked for local government, an (functionally excellent) open source Library system was passed over early in the evaluation stage due to some typo's on their home page that the head librarian picked up on, resulting in a half million dollar purchase of a commercial system. Tragic.

Likewise, if I was working for a large company and I tried to pitch CGRateS to the CTO and upper management, the last release date would be the first thing they would pick up on for sure after they browse the website. What non trivial software does not do releases? Massive red flag.. 

Fortunately I work for a small startup (I believe our CTO (Davey from Vibe Communications in NZ)) met you once in Germany at a conference) so not really a problem advocating it but still, it's another oddity about this project which is confusing enough due to the complexity of the domain.

It seems all the releases are 0.9.1 going back to  2013 so seems like you guys love that release but would urge you in the future to consider semantic versioning.

Doing major releases can also help to gain exposure as bloggers will pick up on it to write about new features, users will reply to posts about rating software saying they just released version x.y.z etc. 

https://www.superlative-solutions.com/blog/ That is one blog I found recently, advocating your product. Good exposure~

Anyway, no need to reply, I see where you are coming from, and thanks again for a great product. I look forward to implementing it in production over the next few weeks.

**Comment by danbogos** [comment]
hey @hampsterx 

I understand the concerns regarding sales and the mentality from  management.
My experience tells me that it does not matter how many good answers you give them, they will still find a reason against it. In the end with opensource it will all reduce to needs and compatibility. Have met quite some number of companies of all sizes who would not buy CGRateS not because of release policy or because of the software quality but because it is cheap! :). So again, I think it is a matter whether they are in need of such solution or not. If they are, they will find the answers and reasons to use it.
If you have been on our google group and have seen our commit history, this should  give you all the reasons you need for using (or not) the software. We are very careful with the quality of the software but we suck seriously on marketing.
Coming back to your questions, I always ask them whether they care about release policy of the closed-source software (ie: Nortel) or whether they know when did Microsoft released Windows last time.
So, bottom line is: we would like to keep commercials out and focus on reasons we think they are worth more: quality, stability and maintainability. Will for sure release more but we need to feel it is right and be ready for it.

In the end, I appreciate you choosing to be a part of the project, so we will be here for you and help on technical issues you will face :).

Have a good one!

DanB
